ci/build_setup.ps1

```diff
@@ -0,0 +1,22 @@
+$ErrorActionPreference = "Stop";
+trap { $host.SetShouldExit(1) }
+
+if ("$env:NUM_CPUS" -eq "") {
+  $env:NUM_CPUS = (Get-WmiObject -class Win32_computersystem).NumberOfLogicalProcessors
+}
+
+if ("$env:ENVOY_BAZEL_ROOT" -eq "") {
+  Write-Host "ENVOY_BAZEL_ROOT must be set!"
+  throw
+}
+
+mkdir -force "$env:ENVOY_BAZEL_ROOT" > $nul
+
+$env:ENVOY_SRCDIR = [System.IO.Path]::GetFullPath("$PSScriptRoot\..")
+
+echo "ENVOY_BAZEL_ROOT: $env:ENVOY_BAZEL_ROOT"
+echo "ENVOY_SRCDIR: $env:ENVOY_SRCDIR"
+
+$env:BAZEL_BASE_OPTIONS="--nomaster_bazelrc --output_base=$env:ENVOY_BAZEL_ROOT --bazelrc=$env:ENVOY_SRCDIR\windows\tools\bazel.rc --batch"
+$env:BAZEL_BUILD_OPTIONS="--strategy=Genrule=standalone --spawn_strategy=standalone --verbose_failures --action_env=HOME --action_env=PYTHONUSERBASE --jobs=$env:NUM_CPUS --show_task_finish $env:BAZEL_BUILD_EXTRA_OPTIONS"
```

Comment:
Does `$HOME` have any value on Windows? (Also in `BAZEL_TEST_OPTIONS` below.)

Reply:
It does not. It doesn't look like we need any of the other `action_env` or `test_env` environment variables on Windows, so we'll remove them as well